checks.d/splunk.py

```diff
@@ -0,0 +1,143 @@
+# stdlib
+import time
+from collections import Counter
+from urlparse import urljoin
+
+# 3rd party
+import requests
+from requests.auth import HTTPBasicAuth
+
+# project
+from checks import AgentCheck
+
+class Splunk(AgentCheck):
+
+    DEFAULT_TIMEOUT = 5
+    CONNECT_CHECK_NAME = 'splunk.can_connect'
+    INDEX_HEALTH_CHECK_NAME = 'splunk.index.is_healthy'
+    PEER_HEALTH_CHECK_NAME = 'splunk.peer.is_healthy'
+
+    def check(self, instance):
+        # if 'url' not in instance:
+        #     raise Exception('Splunk instance missing "url" value.')
+
+        # Load values from the instance config
+        url = instance['url']
+        instance_tags = instance.get('tags', [])
+        default_timeout = self.init_config.get('default_timeout', self.DEFAULT_TIMEOUT)
+        username = self.init_config.get('username')
+        password = self.init_config.get('password')
+        timeout = float(instance.get('timeout', default_timeout))
+
+        self.do_index_metrics(url, username, password, timeout)
+        self.do_peer_metrics(url, username, password, timeout)
+
+    def do_peer_metrics(self, url, username, password, timeout):
+        response = self.get_json(url, '/services/cluster/master/peers', username, password, timeout)
+        for peer in response['entry']:
+            host = peer['content']['label']
+            site = peer['content']['site']
+            searchable = 'true' if peer['content']['is_searchable'] == '1' else 'false'
```

Comment:
`is_searchable` appears to be a JSON boolean (from curl results), is the string `'1'` correct here?